src/main/java/redis/clients/jedis/BinaryClient.java

```diff
@@ -100,9 +102,10 @@ public void set(final byte[] key, final byte[] value) {
     sendCommand(Command.SET, key, value);
   }
 
-  public void set(final byte[] key, final byte[] value, final byte[] nxxx, final byte[] expx,
-      final long time) {
-    sendCommand(Command.SET, key, value, nxxx, expx, toByteArray(time));
+  public void set(final byte[] key, final byte[] value, final byte[] nxxx, final byte[] expx, final byte[] time) {
```

Comment:
@nykolaslima 
Even BinaryClient uses ZParams, so we can change its signature to public void set(final byte[] key, final byte[] value, final SetParams params).
Same things can be applied to BinaryJedis.

@xetorthio @marcosnils What do you think about it?